### PR TITLE
Create sudhirmalik.yml

### PR DESCRIPTION
**Pull request by fatala22**

### CODE REVIEW

**Comment by GuillermoFidalgo** [comment]
I don't like this PR please delete all files.
@fatala22

**Comment by fatala22** [comment]
ok

**Comment by GuillermoFidalgo** [comment]
pre-commit.ci run

**Review by GuillermoFidalgo — CHANGES_REQUESTED** [review]
Please delete this or rename it to a python file with this content


```python

def my_func(x): 
    print(f"input is {x = }")

```

File: malik.yml
Comment (GuillermoFidalgo) [inline malik.yml:1]: This files needs the fix!

**Review by GuillermoFidalgo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**